internal/upload/vmware/vmware.go

```diff
@@ -0,0 +1,23 @@
+package vmware
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func OpenAsStreamOptimizedVmdk(imagePath string) (*os.File, error) {
+	newPath := strings.Replace(imagePath, ".vmdk", "-stream.vmdk", -1)
```

Comment:
```suggestion
	newPath := strings.ReplaceAll(imagePath, ".vmdk", "-stream.vmdk")
```

Reply:
Would
```go
strings.TrimSuffix(imagePath, ".vmdk") + "-stream.vmdk"
```
be even more clear?

Reply:
I agree that this is the best option. Thanks, Lars!